tests/test_symbolics.py

```diff
@@ -620,8 +621,8 @@ def test_minmax_precision(dtype, expected):
     # Check generated code -- ensure it's using the fp64 versions of min/max,
     # that is fminf/fmaxf
     assert all(i in str(op) for i in expected)
-
-    assert np.all(f.data == 6.0)
+    tolerance = 1e-8 if dtype is np.float32 else 1e-16
```

Comment:
I would add tolerance to the parametrization instead of here

Reply:
for all the tests

Reply:
Used numpy inspection instead